### DISCUSSION TITLE
Red Eclipse map : Walls

### DISCUSSION THREAD

**Discussion by Maderasion** [redeclipse/base#1423] [Maps and Content Showcase]
I made a new map for the game , what do you think?
![20230507060636](https://user-images.githubusercontent.com/132848859/236699898-d89c4dc6-0885-4d92-9aac-4ecf817c7feb.png)
![20230507060725](https://user-images.githubusercontent.com/132848859/236699919-77020b0e-183a-4dc7-a00e-9d845b074d5c.png)
![20230507060810](https://user-images.githubusercontent.com/132848859/236699927-01b336fc-0211-46c7-a6cb-e9bb760a76c7.png)
![20230507060900](https://user-images.githubusercontent.com/132848859/236699936-aa86ecce-24f7-4f94-99c1-67b693f9a3b4.png)
![20230507060916](https://user-images.githubusercontent.com/132848859/236699944-2719125e-2b0b-479b-bb4b-e6d9e6f21ce3.png)
![20230507060930](https://user-images.githubusercontent.com/132848859/236699952-2b98e4d9-f81e-4c8e-9cd5-59cb1979f23d.png)

**Comment by dc-redeclipse** [comment]
If you like me to review it and give you some tips, maybe upload your map as an ZIP here and I can look at it.

**Reply by Maderasion** [reply]
I would like you to see the map since I am adding details that I still need to add and perhaps add lighting in certain areas

**Comment by MirceaKitsune** [comment]
The base layout looks alright: Can tell it may need more detail and proper lighting, also the large outer wall the player stands on in the first screenshot should probably be clipped and unreachable. Any plans to post a zip so we can try it out?

**Reply by Maderasion** [reply]
At the moment I am improving the map by adding some details in some areas, with respect to lighting, some areas may have those details on some walls
